persistence/trees/trees_test.go

```diff
@@ -1,19 +1,225 @@
-package trees
+package trees_test
 
-import "testing"
+import (
+	"encoding/hex"
+	"log"
+	"math/big"
+	"testing"
+
+	"github.com/pokt-network/pocket/persistence"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/configs"
+	"github.com/pokt-network/pocket/runtime/test_artifacts"
+	"github.com/pokt-network/pocket/runtime/test_artifacts/keygen"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/messaging"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/shared/utils"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	DefaultChains = []string{"0001"}
+
+	DefaultDeltaBig   = big.NewInt(100)
+	DefaultAccountBig = big.NewInt(1000000)
+	DefaultStakeBig   = big.NewInt(1000000000000000)
+
+	DefaultAccountAmount = utils.BigIntToString(DefaultAccountBig)
```

Comment:
1. Cant find where `DefaultAccountAmount` is used in this file
2. If we do need it, and we need it to be customized for this local test, make it lowercase and update the test
3. If we can use the default, why not just import from `persistence/test/setup_test.go`?

DItto to other variables too.